### PR TITLE
zh-translation: content/zh/docs/setup/install/kubernetes/index.md

### PR DESCRIPTION
**Pull request by jakeslee**
ref: https://github.com/servicemesher/istio-official-translation/issues/1282

### CODE REVIEW

File: content/zh/docs/setup/install/kubernetes/index.md [hunk 1]
```diff
@@ -1,43 +1,38 @@
...
 aliases:
-    - /docs/setup/kubernetes/quick-start/
-    - /docs/setup/kubernetes/install/kubernetes/
+    - /zh/docs/setup/kubernetes/quick-start/
+    - /zh/docs/setup/kubernetes/install/kubernetes/
 ---
 
-This guide installs Istio's built-in `demo` [configuration profile](/docs/setup/additional-setup/config-profiles/).
-This installation lets you quickly evaluate Istio in a Kubernetes cluster on any platform.
+本指南安装使用 Istio 内置的 `demo` [配置文件](/docs/setup/additional-setup/config-profiles/)。
```
Comment (rootsongjc) [inline content/zh/docs/setup/install/kubernetes/index.md]: 链接中需要加 `/zh` 前缀。

**Comment by tanjunchen** [comment]
Please Fix Conflicts.
Once review,I think it can be merged

**Comment by tanjunchen** [comment]
@jakeslee @rootsongjc  原文已经不存在 ， 建议关闭

**Pull request closed without merging**